node/integrationtest/yarn.js

```diff
@@ -131284,7 +131284,7 @@ function isArrayLikeObject(value) {
  * @returns {boolean} Returns `true` if `value` is a buffer, else `false`.
  * @example
  *
- * _.isBuffer(new Buffer(2));
+ * _.isBuffer(Buffer.from(2));
```

Comment:
Again, this is a comment, but the code would throw.

Reply:
I now replaced Buffer.from(<integer>) with Buffer.alloc(<integer>).